packages/arb-token-bridge-ui/src/util/index.ts

```diff
@@ -53,3 +53,23 @@ export const getAPIBaseUrl = () => {
   // Resolves: next-js-error-only-absolute-urls-are-supported in test:ci
   return process.env.NODE_ENV === 'test' ? 'http://localhost:3000' : ''
 }
+
+export enum FeatureFlags {
+  Batch = 'batch'
+}
```

Comment:
just use a string union type, that way you have nice autocomplete when you use it, don't need to import anything else

& nit enum names should be singular

```suggestion
export type FeatureFlag = 'batch'
```